src/connector/src/sink/mod.rs

```diff
@@ -132,6 +132,7 @@ macro_rules! match_sink_name_str {
 
 pub const CONNECTOR_TYPE_KEY: &str = "connector";
 pub const SINK_TYPE_OPTION: &str = "type";
+pub const SINK_WITHOUT_BACKFILL: &str = "without_backfill";
```

Comment:
Personally I perfer calling it `incremental` instead because user may not understand what "backfill" is.

Reply:
Agree. `without` is a privative and `backfill` is a terminology of internal implementation. What about simply calling it `snapshot`?